### PR TITLE
Update Helm Chart with Correct Kubernetes Version Compatible Templates

### PR DESCRIPTION
**Pull request by mbolt35**
Addresses: https://github.com/kubecost/cost-analyzer-helm-chart/issues/590

### CODE REVIEW

**Comment by mbolt35** [comment]
PriorityClass apiVersion template added to address https://github.com/kubecost/cost-analyzer-helm-chart/issues/590

File: cost-analyzer/templates/_helpers.tpl [hunk 1]
```diff
@@ -115,6 +115,50 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 app: cost-analyzer
 {{- end -}}
 
+{{/*
+Return the appropriate apiVersion for daemonset.
+*/}}
+{{- define "cost-analyzer.daemonset.apiVersion" -}}
+{{- if semverCompare "<1.9-0" .Capabilities.KubeVersion.GitVersion -}}
```
Comment (verdverm) [inline cost-analyzer/templates/_helpers.tpl:122]: The `GitVersion` changed `Version` between helm 2 & 3

https://v2.helm.sh/docs/chart_template_guide/

https://helm.sh/docs/chart_template_guide/builtin_objects/

See the `Capabilities` section of each

Comment (mbolt35) [reply cost-analyzer/templates/_helpers.tpl:122]: These templates are lifted directly from the prometheus chart. I imagine we're going to have more issues there as well? https://github.com/kubecost/cost-analyzer-helm-chart/blob/3fc635168847a0c356c8e46d6a88361665f734d6/cost-analyzer/charts/prometheus/templates/_helpers.tpl#L182-L221

Comment (mbolt35) [reply cost-analyzer/templates/_helpers.tpl:122]: Just ran this template with helm 3:

```
kubeVersion: {{ .Capabilities.KubeVersion.GitVersion }}
```

and it generated:
```
kubeVersion: v1.16.0
```

Helm Info:
```bash
~/Repos/kubecost/cost-analyzer-helm-chart$ helm3 version
version.BuildInfo{Version:"v3.1.2", GitCommit:"d878d4d45863e42fd5cff6743294a11d28a9abce", GitTreeState:"clean", GoVersion:"go1.13.8"}
```

It appears this is fine, and backwards compatible.

Comment (verdverm) [reply cost-analyzer/templates/_helpers.tpl:122]: iirc correction, this shows up in Minikube with Helm 3

**Pull request merged**